### ISSUE TITLE
always show the input field even if only use swal('ok');

### ISSUE THREAD

**Issue by killernova** [t4t5/sweetalert#534]
How can I disable the input field?

**Comment by Ammet** [comment]
I have this problem too

**Comment by filipveschool** [comment]
i temporarily fixed it by doing in css: .

.sweet-alert fieldset input[type="text"] {
    display: none;
}

It is maybe not the best solution but at least i dont see the input field now.

**Comment by beastybeast** [comment]
I also have this problem - looks like jquery CSS makes it visible somehow

**Comment by beastybeast** [comment]
The fix from flipveschool do not work for me

**Comment by beastybeast** [comment]
Looks like it is the addition of the class ui-input-text that makes it visible

**Comment by beastybeast** [comment]
Related to: https://github.com/t4t5/sweetalert/issues/351

**Comment by beastybeast** [comment]
The temporary fix for me is just to call

`$(".sweet-alert" ).remove();`

before I show the dialog!

**Comment by amitpatil321** [comment]
I am also facing this thing, dont know from where its getting added. In my case below code is getting added.
```
<div class="form-group">
      <input type="text" tabindex="3" class="form-control" placeholder="">
      <span class="sa-input-error help-block">
        <span class="glyphicon glyphicon-exclamation-sign"></span> <span class="sa-help-text">Not valid</span>
      </span>
</div>
```

**Comment by nfriend21** [comment]
If you look on sweetalert's site and inspect their confirmation messages that dont have the input displaying, you will see that the input actually still is there in the code, there is just default CSS from sweetalert that hides it.  Then, if you inspect the code on your own site, you likely have CSS that is overriding it with a "display: block;".  That was the problem in my case.

**Comment by t4t5** [comment]
This is fixed in SweetAlert 2.0 since it build every modal from scratch instead of showing/hiding different elements. You have to use `content: "input"` though. Make sure you read the [upgrade guide](https://sweetalert.js.org/guides/#upgrading-from-1x) to see all the new changes!

**Comment by sarphu** [comment]
is there a property or method in sweetalert to prefill text in input.
